handle-results.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python3
+import pathlib
+import sys
+import pandas as pd
+import argparse
+
+# All time measurements are in seconds
+
+# rate suite not supported yet
+# Reference times taken from https://www.spec.org/cpu2017/results/res2017q2/cpu2017-20161026-00003.html
```

Comment:
What was the reasoning to compare against an UltraSPARC vs something in-house? Was the assumption that someone will just make better plots out of the data later and this is just a placeholder?

Reply:
This is the official spec score baseline as defined by the org. When reporting SPEC numbers, you're supposed to compare to that machine for apples-to-apples comparison.